tests/integration/test_job_runner_mgr.py

```diff
@@ -82,3 +83,58 @@ async def test_kill_error(one, start, test_dir, capsys, log_filter):
             level=logging.WARNING,
         )
         assert itask.state(TASK_STATUS_RUNNING)
+
+
+async def test_create_nn_new(one, start, test_dir, capsys, log_filter):
+    """Test _create_nn. 
+
+    It should create the NN symlink.
+    """
+    async with start(one):
+        itask = one.pool.get_tasks()[0]
+
+        workflow_job_log_dir = Path(get_workflow_run_job_dir(one.workflow))
+        job_id = itask.tokens.duplicate(job='01').relative_id
+        job_log_dir = Path(workflow_job_log_dir, job_id)
+        job_log_dir.mkdir(parents=True)
+
+        # call _create_nn
+        JobRunnerManager()._create_nn(job_log_dir / 'job.out')
+
+        # check the symlink exists
+        assert (job_log_dir.parent / "NN").is_symlink()
+
+
+async def test_create_nn_old(one, start, test_dir, capsys, log_filter):
+    """Test _create_nn.
+
+    It should remove existing job logs, if the dir already exists.
+    """
+    async with start(one):
+        itask = one.pool.get_tasks()[0]
+
+        # fake some old job logs
+        workflow_job_log_dir = Path(get_workflow_run_job_dir(one.workflow))
+        job_id = itask.tokens.duplicate(job='01').relative_id
+        job_log_dir = Path(workflow_job_log_dir, job_id)
+        job_log_dir.mkdir(parents=True)
+
+        job_logs = []
+        for name in JOB_LOG_OUT, JOB_LOG_ERR:
+            job_logs.append(job_log_dir / name)
+
+        # create the logs
+        for job_log in job_logs:
+            job_log.touch()
+
+        # check they exist
+        for job_log in job_logs:
+            assert job_log.is_file()
+
```

Comment:
Surely not needed? 😅
```suggestion
```

Reply:
Fair point, given the creation method (originally I tried to create them via Cylc methods)